app/models/User.coffee

```diff
@@ -524,6 +524,7 @@ module.exports = class User extends CocoModel
   useGoogleAnalytics: -> not (features?.chinaInfra ? false)
   # This flag is set globally for our China server.
   showChinaVideo: -> features?.china ? false
+  canAccessCampaignFreelyFromChina: (campaignID) -> campaignID == "55b29efd1cd6abe8ce07db0d" or campaignID == "5789236960deed1f00ec2ab8" or campaignID == "578913f2c8871ac2326fa3e4"
 
```

Comment:
I think this is all I can move to user model

Reply:
This is good.